pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.saucelabs.bamboo</groupId>
     <artifactId>bamboo-sauceondemand-plugin</artifactId>
-    <version>1.6.78-SNAPSHOT</version>
```

Comment:
My intent was to call out the one-off-ness of this build. If we do a `1.6.79-SNAPSHOT` build I would think we want some way of denoting that it's different from the regular versions before/after it? Where would the better place to do that be?